website/build.gradle

```diff
@@ -104,3 +106,32 @@ task testWebsite(type: Exec) {
 task preCommit {
   dependsOn testWebsite
 }
+
+task mergeWebsite << {
+  /* Disabled for now, for testing
+  exec {
+    executable 'git'
+    args 'checkout', 'asf-site'
+  }
+  copy {
+    from contentBuildDir
+    into contentRepoDir
+  }
+  exec {
+    executable 'git'
+    args 'add', contentRepoDir
+  }
+  exec {
+    executable 'git'
+    args 'commit', '-m', 'Update website ' + new Date().format('yyyy/MM/dd HH:mm:ss')
```

Comment:
If it's easy to grab, it would be useful to also include the master commit hash in the commit message. (If it's not easy, then don't worry about it)